### PR TITLE
Ssk/make readonly

### PR DESCRIPTION
**Pull request by shaswa**
**Why This PR?**
This PR makes kube-hunter write to the mounted FS instead of the root FS 

**Changes**
Changes proposed in this pull request:

* Added `securityContext ` to kube-hunter 

**Checklist:**

* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x]  Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or helm-docs has been run for the charts that support it.

### CODE REVIEW

File: stable/insights-agent/templates/kube-hunter/_job.yaml [hunk 1]
```diff
@@ -25,6 +27,12 @@ containers:
   volumeMounts:
   - name: output
     mountPath: /output
+  - name: tmp
+    mountPath: /tmp
```
Comment (makoscafee) [inline stable/insights-agent/templates/kube-hunter/_job.yaml:31]: why did we mount`/temp` here?

**Review by makoscafee — APPROVED** [review]
LGTM. just had one question

File: stable/insights-agent/templates/kube-hunter/_job.yaml (hunk 1 above)
Comment (baderbuddy) [reply stable/insights-agent/templates/kube-hunter/_job.yaml:31]: In Trivy we had to do the same thing. kube-hunter must write files to /tmp, and when we make the root filesystem read only we can't write to tmp anymore if it's part of the root filesystem. But by creating an emptyDir volume (which is writeable) and mounting it to /tmp, then /tmp is writeable again.

**Review by baderbuddy — APPROVED** [review]
This looks good to me.

**Comment by shaswa** [comment]
Thanks you, Barnie and Bader!

File: stable/insights-agent/templates/kube-hunter/_job.yaml (hunk 1 above)
Comment (makoscafee) [reply stable/insights-agent/templates/kube-hunter/_job.yaml:31]: makes sense

**Pull request merged**